pkg/scheduler/ugm/queue_tracker.go

```diff
@@ -266,6 +209,15 @@ func (qt *QueueTracker) headroom(hierarchy []string, trackType trackingType) *re
 		if trackType == user {
 			if config := m.getUserWildCardLimitsConfig(qt.queuePath); config != nil {
 				headroom = config.maxResources.Clone()
+				log.Log(log.SchedUGM).Debug("Use wild card limit settings as there is no limit set explicitly",
```

Comment:
Wildcard use implicitly means no limit set. No need to add that to the message.
We also should only need this check as part of the create of a new queue tracker. After that the config change should be setting or removing the value. We should not need it at all at this point.